.github/workflows/holonix-cache.yml

```diff
@@ -67,6 +63,9 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        target:
+          - ".#devShells.{0}.holonix"
+          - ".#packages.{0}.hc-scaffold"
         extra_args:
           - '--override-input versions ./versions/0_1'
```

Comment:
Let's find out

Reply:
The Rust code didn't properly handle empty inputs but otherwise this should work. I've updated the Rust code here https://github.com/holochain/nix-cache-check/commit/240802b34da3af50fa409505e180f89e9175662f

Reply:
Hmm, it's not refreshing the commit it's pulling for the v1 tag...